BRANDING.MD

```diff
@@ -0,0 +1,24 @@
+## Knative Branding Guidelines
+
+
+The Knative logo is made available here with the following guidelines:
+
+ **You may:**
+ - use the logo for non-commercial uses such as t-shirts and stickers.
+ - use the logo to link to knative.dev
+ - use the logo in a blog post or news article about Knative
+ - use the logo to advertise that your product has support for Knative [Note that at this time such usage does *not* indicate conformance with any standards or requirements]
+
+**You may not:**
+
+ - Use the Knative logo as your application's icon or logo
+ - Create a modified version of the Knative logo (except for non-commercial uses such as t-shirts and stickers)
+ - Integrate the Knative logo into your logo
+ - Change the Knative logo's colors
+ - Use the Knative logo in such a manner as to create the impression that your product or service has been certified, vetted, or otherwise endorsed by the Knative project. 
```

Comment:
<!--whitespace-bot-->
Remove trailing whitespace:
```suggestion
 - Use the Knative logo in such a manner as to create the impression that your product or service has been certified, vetted, or otherwise endorsed by the Knative project.
```